### PR TITLE
[WNMGDS-2238] Healthcare privacy settings modal button css cascading incorrectly.

### PR DESCRIPTION
**Pull request by forrestbaer**
## Summary

- updates hover color for privacy settings button in healthcare footer to match other links
- updates cascade to only style the first child button in this case so it doesn't override button in privacy settings modal

## How to test

1. `yarn clean && yarn install && yarn build && yarn storybook`
2. Open [http://localhost:6006/?path=/story/healthcare-footer--basic-footer&globals=theme:healthcare](http://localhost:6006/?path=/story/healthcare-footer--basic-footer&globals=theme:healthcare)
3. Inspect privacy settings modal button, confirm it does not include reset styles from `.hc-c-footer__inline-item > button`

## Checklist

- [x] Prefixed the PR title with the [Jira ticket number](https://jira.cms.gov/projects/WNMGDS/) as `[WNMGDS-####] Title` or [NO-TICKET] if this is unticketed work.
- [x] Selected appropriate `Type` (only one) label for this PR, if it is a breaking change, label should only be `Type: Breaking`
- [x] Selected appropriate `Impacts`, multiple can be selected.

### CODE REVIEW

File: packages/ds-healthcare-gov/src/styles/components/_Footer.scss [hunk 1]
```diff
@@ -43,8 +43,9 @@
     padding: 0;
     text-decoration: none;
 
+    &:hover,
     &:focus {
-      @include mixins.focus-styles;
+      color: var(--color-primary-darker);
```
Comment (pwolfert) [inline packages/ds-healthcare-gov/src/styles/components/_Footer.scss:48]: Why was this change made to focus? What was there before wasn't perfect either, but this is what it looks like now:

focus

**Review by pwolfert — APPROVED** [review]
Nice and concise fix 👍
Question below about the other change

File: packages/ds-healthcare-gov/src/styles/components/_Footer.scss (hunk 1 above)
Comment (forrestbaer) [reply packages/ds-healthcare-gov/src/styles/components/_Footer.scss:48]: Oh shoot didn't notice the focus state, I'll make that only on hover. The hover state previously is grey and should be blue.

Comment (forrestbaer) [reply packages/ds-healthcare-gov/src/styles/components/_Footer.scss:51]: Updated a couple things here to resolve this
image

Comment (pwolfert) [reply packages/ds-healthcare-gov/src/styles/components/_Footer.scss:51]: Nice, thank you 👍

**Pull request merged**